.all-contributorsrc

```diff
@@ -236,6 +236,44 @@
       "contributions": [
         "code"
       ]
+    },
+    {
+      "login": "vandana1499",
+      "name": "unbeat",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/29394600?v=4",
+      "profile": "https://github.com/vandana1499",
+      "contributions": [
+        "doc"
+      ]
+    },
+    {
+      "login": "lARSHADl",
+      "name": "Arshad Ahmed",
+      "avatar_url": "https://avatars3.githubusercontent.com/u/45604332?v=4",
+      "profile": "https://github.com/lARSHADl",
+      "contributions": [
+        "doc"
+      ]
+    },
+    {
+      "login": "Ergouzii",
+      "name": "Xiaohui Liu",
+      "avatar_url": "https://avatars1.githubusercontent.com/u/33507446?v=4",
+      "profile": "http://xiaohuiliu.me",
+      "contributions": [
+        "doc",
+        "code",
+        "design"
+      ]
+    },
+    {
+      "login": "Seungyeon-Lee",
+      "name": "Seungyeon-Lee",
+      "avatar_url": "https://avatars1.githubusercontent.com/u/26589915?v=4",
+      "profile": "https://seungyeon-lee.github.io/",
+      "contributions": [
+        "code"
+      ]
```

Comment:
Why is this file in this? If it a git related thing I suggest you to 
- make a new branch from master and check into it.
```sh
git checkout -b <new-branch-name> saadpasta/master
```
if saadpasta is not know to git than
```sh
git remote add saadpasta https://github.com/saadpasta/developerFolio.git
```
and then run previous command.

- Squash and merge this branch into that new branch
```sh
git merge --squash origin feat/dark-mode
```
- Commit it and force push to this branch.
```sh
git commit -m "Added Dark Mode Feature"
git push -f origin <new branch name>:feat/dark-mode
```

This would remove all of this issue. But it will remove all previous code review which isn't a worry for me. @kartikcho or @saadpasta do you have any objection in doing that?

Reply:
That _should_ fix all the issues but seems like an unconventional way of dealing with this.
 
Wouldn't merging into the new branch still show these changes? Because this branch is on par with master too so the new branch with all the changes from here would also be dealing with the same @naveen521kk.

Reply:
> Wouldn't merging into the new branch still show these changes? Because this branch is on par with master too so the new branch with all the changes from here would also be dealing with the same 

I don't think so. Doing those steps would just mean that we have a single commit on this whole PR and online diff between master and this is shown. Pretty much it is "unconventional " but actually far easier than rebasing the whole branch with the master which may create some issue not rebased properly.

Reply:
I have already rebased with master btw.

Reply:
@MuhammadHasham23 Looks like you have rebased the branch with the master branch on your forked repo you need to rebase master with developerFolio master. 
Try reading this 
https://medium.com/@topspinj/how-to-git-rebase-into-a-forked-repo-c9f05e821c8a

Reply:
@MuhammadHasham23 Let's complete this PR before the end of this month ✌✌

Reply:
Yeah sure, I'll wrap it up tomorrow (Saturday) @saadpasta

Reply:
@saadpasta I did rebased with the upstream branch (i.e; from the original repo).